src/Metadata/Extractor/YamlExtractor.php

```diff
@@ -32,7 +32,7 @@
     protected function extractPath(string $path)
     {
         try {
-            $resourcesYaml = Yaml::parse(file_get_contents($path));
+            $resourcesYaml = Yaml::parse(file_get_contents($path), Yaml::PARSE_KEYS_AS_STRINGS);
```

Comment:
Hmm why this change? Did you hit a bug? This might need to be backported to 2.0 if it's the case.